### ISSUE TITLE
Source code not up to date

### ISSUE THREAD

**Issue by Thundernerd** [Root-MtX/Nro2Nsp#51]
The source code builds version 3.2.2 while the latest stable version is 3.3.5
Any plans on updating the source code?

**Comment by drizzt** [comment]
What do you mean? The last pushed tag is 3.3.6Beta4

**Comment by Thundernerd** [comment]
That's right. The pre-compiled version is 3.3.6Beta4. However, the C# source code (in the Source folder) is from version 3.2.2 and hasn't been updated in two years according to GitHub.
Are you planning to push the C# source code to the repository as well?

**Comment by Thundernerd** [comment]
@Root-MtX any update on this? If the source code is up to date then we can add nice features ourselves :)
